runtime/autoload/provider/node.vim

```diff
@@ -22,6 +22,30 @@ function! s:is_minimum_version(version, min_major, min_minor) abort
     \         && str2nr(v_list[1]) >= str2nr(a:min_minor)))
 endfunction
 
+function! s:get_node_client_entry(package_manager) abort
+  if !executable(a:package_manager)
+    return ''
+  endif
+  let cli_bin = '/neovim/bin/cli.js'
+  let is_yarn = a:package_manager ==# 'yarn'
+  let cmd = is_yarn ? 'yarn global dir' : 'npm root -g'
+  let modules_dir = get(split(system(cmd), "\n"), 0, '')
+  if v:shell_error || !isdirectory(modules_dir)
+    return ''
+  endif
+  " `yarn global dir` returns the root directory of the global modules
+  " so we need to append the `/node_modules` part
+  let global_modules_dir = is_yarn ? modules_dir . '/node_modules' : modules_dir
+  if !isdirectory(global_modules_dir)
+    return ''
+  endif
+  let entry_path = glob(global_modules_dir . cli_bin)
```

Comment:
`cli_bin` is used once so why not `glob(global_modules_dir . '/neovim/bin/cli.js')`?